### ISSUE TITLE
CommentsIndentation Check false-positive when some statement are used without curly braces on the same line

### ISSUE THREAD

**Issue by Vladlis** [checkstyle/checkstyle#3125]
```
vlad@vlad-N61PC-M2S ~/cli $ java -jar checkstyle-6.17-all.jar com.puppycrawl.tools.checkstyle.Main -c my_check.xml Test.java
Starting audit...
[ERROR] /home/vlad/cli/Test.java:5: Comment has incorrect indentation level 8, expected is 16, indentation should be the same level as line 4. [CommentsIndentation]
Audit done.
Checkstyle ends with 1 errors.
vlad@vlad-N61PC-M2S ~/cli $ cat Test.java
public class Test {
    void foo1() {
        if (true) System.out.println(
                "some string"); // line 4
        // violation - line 5
    }
}
vlad@vlad-N61PC-M2S ~/cli $ cat my_check.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">


<module name = "Checker">
    <module name="TreeWalker">
         <module name="CommentsIndentation">
         </module>
    </module>
</module>
```
Expected - no violations.

**Comment by Vladlis** [comment]
The issue does not reproduce with the latest snapshot.
```
~/cli$ java -jar checkstyle-7.0-SNAPSHOT-all.jar -c my_check.xml Test.java
Starting audit...
Audit done.
~/cli$ cat Test.java
public class Test {
    void foo1() {
        if (true) System.out.println(
                "some string");
        // comment
    }
}
~/cli$ cat my_check.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">


<module name = "Checker">
    <property name="charset" value="UTF-8"/>
    <property name="severity" value="warning"/>
    
    <module name="TreeWalker">
         <module name="CommentsIndentationCheck"/>
    </module>
</module>
```
